### ISSUE TITLE
Connect with ssh

### ISSUE THREAD

**Issue by gerw** [Bouni/luxtronik-firmware-analysis#7]
On my system, I also have a fairly recent version of `ssh`. However, I still can connect with
```
ssh -oKexAlgorithms=+diffie-hellman-group1-sha1  -oCiphers=aes128-cbc -oHostKeyAlgorithms=+ssh-rsa root@heatpump
```

Maybe, this also works for other people?

**Comment by gerw** [comment]
Even more convenient, one could add the following to `.ssh/config`:
```
Host heatpump
	User root
	KexAlgorithms diffie-hellman-group1-sha1
	HostKeyAlgorithms ssh-rsa
	Ciphers aes128-cbc
```
Now, I can connect with simply `ssh heatpump`.

**Comment by Bouni** [comment]
I just tried to verify this with my Arch system that has `OpenSSH_9.9p1, OpenSSL 3.4.0 22 Oct 2024` and it failed with `port 22: Connection refused`.

I found that very strange and then realized, the SSH port is no longer open:

```sh
sudo nmap -p- 192.168.0.123
Starting Nmap 7.95 ( https://nmap.org ) at 2025-02-03 13:37 CET
Nmap scan report for wp.home (192.168.0.123)
Host is up (0.00030s latency).
Not shown: 65533 closed tcp ports (reset)
PORT      STATE SERVICE
8889/tcp  open  ddi-tcp-2
36000/tcp open  unknown
MAC Address: 40:EC:F8:07:2D:43 (Siemens AG)

Nmap done: 1 IP address (1 host up) scanned in 2.57 seconds
```

I recently installed the latest firmware `V3.90.3` via the Cloud portal out of curiosity and it seems that they used that to close the SSH port ...

**Comment by gerw** [comment]
Yes, this feature was considered to be a bug, see https://www.heise.de/news/Waermepumpen-von-Alpha-Inntotec-und-Novelan-mit-hardkodiertem-Passwort-9618846.html.

It was removed in version V3.90.0, I think. Now, `appl` kills the ssh server within the first second of its life.

**Comment by Bouni** [comment]
I totally missed that. Anyway not that much that could have been done via ssh
